app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,8 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find_by(id: params[:user_id])
+    @user = User.includes(:posts).find_by(id: params[:user_id])
     @posts = @user.posts
+    # @posts = Post.includes(:author).find_by(author_id: params[:user_id])
   end
 
```

Comment:
- [ ] Kindly remove the commented-out code in the `Posts#index` action. Removing such codes makes your work clean and readable. 👍 💻